### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by annextv** [scivision/PyLivestream#94] [Q&A]
Hi,

In a scenario that I want to run multiple instances at the same time (using screen) what setting do I need to change since there is only one pylivestream.ini

I want to broadcast to 3 youtube channels at the same time (3 different videos) and/or to the same channel 3 different videos by using 3 different keys. Can you please tell me what I need to change in order to get that result? How can I get the program to read the specs, for example, from 3 different pylivestream.ini files?

Thanks in advance 

BTW it works perfectly!

**Comment by tabanger** [comment]
Just create 3 separate sections in your pylivestream.ini file.  For example, [youtube1], [youtube2], and [youtube3].

**Comment by scivision** [comment]
In the new PyLivestream >= 2.0, just copy multiple JSON file and edit from example: https://github.com/scivision/PyLivestream/blob/main/src/pylivestream/data/pylivestream.json
